src/skan/test/test_csr.py

```diff
@@ -179,6 +179,62 @@ def test_transpose_image():
             )
 
 
+@pytest.mark.parametrize(
+        "skeleton,prune_branch,target",
+        [
+                (
+                        skeleton1, 1,
+                        np.array([[0, 1, 1, 1, 1, 1, 0], [1, 0, 0, 0, 0, 0, 1],
+                                  [0, 1, 1, 0, 1, 1, 0], [0, 0, 0, 1, 0, 0, 0],
+                                  [0, 0, 0, 0, 0, 0, 0]])
+                        ),
+                (
+                        skeleton1, 2,
+                        np.array([[0, 0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0, 0],
+                                  [0, 1, 0, 0, 0, 0, 0], [1, 0, 0, 2, 0, 0, 0],
+                                  [1, 0, 0, 0, 2, 2, 2]])
+                        ),
+                # There are no isolated cycles to be pruned
+                (
+                        skeleton1, 3,
+                        np.array([[0, 1, 1, 1, 1, 1, 0], [1, 0, 0, 0, 0, 0, 1],
+                                  [0, 3, 2, 0, 1, 1, 0], [3, 0, 0, 4, 0, 0, 0],
+                                  [3, 0, 0, 0, 4, 4, 4]])
+                        ),
+                ]
```

Comment:
Eek, looks like I need to fiddle with my yapf config a bit more, I very much dislike this way of closing/matching parens... 😅  (Ignore this comment for this PR, just thinking out loud.)